test/HyperliquidInterchainToken.js

```diff
@@ -4,14 +4,18 @@ const { ethers } = require('hardhat');
 const {
     constants: { AddressZero },
     getContractAt,
-    utils: { keccak256 },
+    utils: { keccak256, toUtf8Bytes },
     provider,
 } = ethers;
 const { expect } = require('chai');
 const { getRandomBytes32, getEVMVersion } = require('./utils');
 const { deployAll } = require('../scripts/deploy');
 const { ITS_HUB_ADDRESS } = require('./constants');
 
+function getDeployerStorageSlot() {
```

Comment:
avoid get prefixes lol
```suggestion
function deployerStorageSlot() {
```